common-custom-user-data-maven-extension/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+## [1.2] - 2020-08-31
+### Added
+- Add support for TravisCI
+- Add support for tagging IDEA based builds
+
+### Removed
+- Remove build scan tag for Maven version
```

Comment:
...since Gradle Enterprise 2020.3+ allows filtering by Maven version out-of-the-box

Reply:
Adapted.